Assignment1/src/test/java/files/Utils.java

```diff
@@ -1,14 +1,10 @@
-
-// request spec builders
-// response spec builders
 package files;
 
 import io.restassured.path.json.JsonPath;
 
 public class Utils {
-		
-	 public static JsonPath rawToJson(String response) {
-		 JsonPath js = new JsonPath(response);
-		 return js;
-	 }
+	public static JsonPath rawToJson(String response) {
+		JsonPath js = new JsonPath(response);
+		return js;
+	}
 }
```

Comment:
Class files are binary files and should not be made a part of the PR. You repository should only contain code and configuration required setup and run the code.

Reply:
Zaheer Whatsapp comment - The comment is meant for all the .class files. Not this particular java class. Java compiles code into byte code adn puts it in these files. They are not code so should not be in a repository.  Since I couldn't leave the comment on a binary file (github can't show a diff for them) I left comment on an adjacent file.